src/control/drpc/property.go

```diff
@@ -79,6 +87,27 @@ func LabelIsValid(label string) bool {
 	return bool(C.daos_label_is_valid(cLabel))
 }
 
+func EcCellSizeIsValid(sz uint64) bool {
+	if sz > math.MaxUint32 {
+		return false
+	}
+	return bool(C.daos_ec_cs_valid(C.uint32_t(sz)))
+}
+
+func EcPdaIsValid(pda uint64) bool {
+	if pda > math.MaxUint32 {
+		return false
+	}
+	return bool(C.daos_ec_pda_valid(C.uint32_t(pda)))
+}
+
+func RpPdaIsValid(pda uint64) bool {
```

Comment:
Since these are exported methods in Go (start with a capital letter), you should add a comment for each starting with the name of the function, similar to what was done for the LabelIsValid function above.